src/TurbulenceClosures/turbulence_closure_implementations/scalar_diffusivity.jl

```diff
@@ -35,10 +35,20 @@ Arguments
 Keyword arguments
 =================
 
-`ν` and the fields of `κ` may be constants (converted to `FT`), arrays, fields or
-  - functions of `(x, y, z, t)` if `discrete_form = false`
-  - functions of `(i, j, k, grid, LX, LY, LZ)` with `LX`, `LY` and `LZ` are either `Face()` or `Center()` if
-    `discrete_form = true`.
+* `ν`: Viscosity. `Number`, three-dimensional `AbstractArray`, `Field`, or `Function`.
+
+* `κ`: Diffusivity. `Number`, `AbstractArray`, `Field`, `Function`, or
+       `NamedTuple` of diffusivities with entries for each tracer.
+
+* `discrete_form`: `Boolean`; default: `False`.
+
+When prescribing the viscosities or diffusivities as functions, depending on the value of keyword argument
+`discrete_form`, the constructor expects:
+
+  - `discrete_form = false` (default): functions of the grid's native coordinates and time, e.g., `(x, y, z, t)` for a
+    `RectilinearGrid` or `(λ, φ, z, t)` for a `LatitudeLongitudeGrid`.
+
+  - `discrete_form = true`: functions of `(i, j, k, grid, ℓx, ℓy, ℓz)` with `ℓx`, `ℓy` and `ℓz` either `Face()` or `Center()`.
```

Comment:
I've never used the discrete form so I'm not sure how it works here, but how do the `ℓx`, `ℓy` and `ℓz` arguments work? Do we have consider all face combinations when providing the function? Should this be also explained here?

Reply:
You need to prescribe where you want the diffusivity to be computed. That’s what I understand. 

What do you mean by “consider all combinations”? Yes, all 8 combinations are in principle allowed. Is this what you mean?

Reply:
> What do you mean by “consider all combinations”? Yes, all 8 combinations are in principle allowed. Is this what you mean?

I mean more as is: are all combinations _required_ the be defined?

Reply:
Nah. Only the one you want.

Reply:
OK, I see what you mean! I added an example which I think clarifies it?